Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -405,6 +405,8 @@ def __init__(self, scatter_widget, parent=None, view_box=ViewBox):
         self.plot_widget.scene().installEventFilter(self._tooltip_delegate)
         self.view_box.sigTransformChanged.connect(self.update_density)
 
+        self._timer = QTimer()
```

Comment:
I'd set it to `None`. And then add checks for `None`, e.g `if self._timer is not None and self._timer.isActive():`.